fe/fe-core/src/main/java/com/starrocks/datacache/LoadDataCacheMetrics.java

```diff
@@ -0,0 +1,102 @@
+// Copyright 2021-present StarRocks, Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.starrocks.datacache;
+
+import com.starrocks.monitor.unit.ByteSizeValue;
+import com.starrocks.monitor.unit.TimeValue;
+import com.starrocks.thrift.TLoadDataCacheMetrics;
+
+import java.util.concurrent.TimeUnit;
+
+public class LoadDataCacheMetrics {
+    private final ByteSizeValue readBytes;
+    private final TimeValue readTimeNs;
+    private final ByteSizeValue writeBytes;
+    private final TimeValue writeTimeNs;
+
+    private final DataCacheMetrics lastDataCacheMetrics;
+
+    private LoadDataCacheMetrics(ByteSizeValue readBytes, TimeValue readTimeNs, ByteSizeValue writeBytes,
+                                 TimeValue writeTimeNs,
+                                 DataCacheMetrics lastDataCacheMetrics) {
+        this.readBytes = readBytes;
+        this.readTimeNs = readTimeNs;
+        this.writeBytes = writeBytes;
+        this.writeTimeNs = writeTimeNs;
+        this.lastDataCacheMetrics = lastDataCacheMetrics;
+    }
+
+    public static LoadDataCacheMetrics mergeMetrics(LoadDataCacheMetrics before, LoadDataCacheMetrics now) {
+        ByteSizeValue mergedReadBytes =
+                new ByteSizeValue(before.getReadBytes().getBytes() + now.getReadBytes().getBytes());
+        TimeValue mergedReadTimeNs = new TimeValue(before.getReadTimeNs().nanos() + now.getReadTimeNs().nanos(),
+                TimeUnit.NANOSECONDS);
+        ByteSizeValue mergedWriteBytes =
+                new ByteSizeValue(before.getWriteBytes().getBytes() + now.getWriteBytes().getBytes());
+        TimeValue mergedWriteTimeNs =
```

Comment:
It is strange to accumulation all BE latency as the write cache latency, it usually is far greater than the whole warmup task time and may be confusing.